src/data/error.js

```diff
@@ -0,0 +1,29 @@
+import { writable } from 'svelte/store'
+
+const defaultError = {
+  code: 0,
+  message: '',
+}
+
+export const error = writable(defaultError)
+
+init()
+
+function init() {
+  window.onunhandledrejection = ({reason}) => {
+    error.set(reason)
```

Comment:
We might want to rename `reason` to something that communicates that it will have "code" and "message" attributes (like our `defaultError` above).